03-single-hop-ipv6-icmp/03-single-hop-ipv6-icmp.md

```diff
@@ -62,16 +63,18 @@ host, e.g. Linux host) simultaneously to one native node for 1 hour.
 All nodes are still running, reachable, and the packet buffer is empty 10
 seconds after completions (use module `gnrc_pktbuf_cmd`).
 Packet loss is irrelevant in this stress test.
+For regression sake currently the 0ms ping6 has `~10% packet loss`.
```

Comment:
No. The stress tests are not about packet loss, but the ability of the node to handle heavy load. That is why the packet loss is irrelevant. Putting a number to it puts unrealistic expectations into our constrained system.